requirements.d/development.txt

```diff
@@ -1,4 +1,4 @@
-virtualenv<14.0
+virtualenv==16.7.7
 tox<3.10
 pytest<5.0
```

Comment:
did you check these 2?

could be that they are only there because of py34 or trusty.

Reply:
Good point. For justifications on those versions:

**PyTest:** Python 3.5+ is indeed the restriction, https://docs.pytest.org/en/latest/changelog.html#pytest-5-0-0-2019-06-28

**Tox:** Maybe due to Tox inheriting from local Virtualenv (see: https://tox.readthedocs.io/en/latest/changelog.html#v3-10-0-2019-05-13), and potentially requiring a lower version of Virtualenv, mentioned in this commit: 3b54da2

EOL for 3.5 is Sept. 2020 per [here](https://devguide.python.org/#status-of-python-branches), so we probably don't need to worry (too much) about version breakage for a while. Going to remove the version locks on these (as well as virtualenv) for the requirements.txt, but maintain those on the .travis/install.sh just in case things change.

Going to make that update now - will see if things pass!